oms-azure-vminventory-solution/azuredeploy.json

```diff
@@ -0,0 +1,1503 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "omsLogAnalyticsWorkspaceName": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Create new or use an existing Log Analytic Workspace"
+      }
+    },
+    "omsLogAnalyticsRegion": {
+      "type": "string",
+      "defaultValue": "eastus",
+      "allowedValues": [
+        "westeurope",
+        "eastus",
+        "southeastasia",
+        "Australia Southeast",
+        "wescentralus"
+      ],
+      "metadata": {
+        "description": "Specify the Azure Region for your new or existing OMS workspace"
+      }
+    },
+    "omsLogAnalyticsSku": {
+      "type": "string",
+      "defaultValue": "free",
+      "allowedValues": [
+        "free",
+        "standalone",
+        "pernode"
+      ],
+      "metadata": {
+        "description": "Specify the SKU for Log Analytics"
+      }
+    },
+    "omsAutomationAccountName": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "Use an existing Automation account or create a new"
+      }
+    },
+    "omsAutomationRegion": {
+      "type": "string",
+      "defaultValue": "eastus2",
+      "allowedValues": [
+        "westeurope",
+        "southeastasia",
+        "eastus2",
+        "southcentralus",
+        "japaneast",
+        "southeastasia",
+        "southcentralus",
+        "northeurope",
+        "canadacentral",
+        "australiasoutheast",
+        "centralindia",
+        "japaneast"
+      ],
+      "metadata": {
+        "description": "Specify the Azure Region for your OMS Automation Account"
+      }
+    },
+    "omsAutomationSKU": {
```

Comment:
be consistent. You have omsLogAnalyticsSku - and omsAutomationSKU. Please change to omsAutomationSku